release_notes/topics/20_0_0.adoc

```diff
@@ -19,24 +43,56 @@ In this release, we are making important changes to `kc.bat` to give the same ex
 
 = Upgrade of embedded H2 database
 
-{project_name} ships for development purposes with an H2 database driver. As it is intended for development purposes only, it should never be used in a production environment.
+{project_name} ships for development purposes with an H2 database driver. As it is intended for development purposes
+only, it should never be used in a production environment.
 
 In this release, the H2 driver has been upgraded from version 1.x to version 2.x.
-See link:{upgradingguide_link}[{upgradingguide_name}] for details on how to migrate.
 
-= New Keycloak Operator upgrade
+= Feature guard for hosting the Keycloak JavaScript adapter
 
-We are happy to announce that the new Quarkus-based Keycloak Operator is no longer a tech preview feature. We added new functionality as well as the quality of life improvements. Some of them resulted in breaking changes.
+Applications are able to load `keycloak.js` directly from the Keycloak server. As it's not considered a best-practice
+to load JavaScript libraries this way there is now a feature guard that allows disabling this ability.
 
-It’s important to emphasize that none of these breaking changes will come into effect automatically. Please, read our link:{upgradingguide_link}[{upgradingguide_name}] for more information.
+In Keycloak 21 we will deprecate this option, and in Keycloak 22 we plan to completely remove the ability to load
+`keycloak.js` from the Keycloak server.
 
-== Realm Operator
+= OTP Application SPI
+
+In previous releases the list of OTP applications displayed to users was hard-coded in Keycloak. With the introduction of
+the OTP Application SPI it is now possible to disable built-in OTP applications, as well as adding custom OTP Applications.
+
+= Custom Identity Providers can now set an icon for the provider
+
+A custom identity provider can now set the icon used on the login pages. Thanks to https://github.com/klausbetz[Klaus Betz],
+who happens also to maintain
+https://github.com/klausbetz/apple-identity-provider-keycloak[an extension to Keycloak to support log in with AppleID].
+
+= Open Banking UK support
+
+Keycloak is now compliant with Open Banking UK. Details in the link:{adapterguide_link}#_open-banking-uk[{adapterguide_name}].
```

Comment:
This link with the details is not available in this PR. It requires additional changes in the file "fapi-support.adoc", which I've included in the other PR: https://github.com/keycloak/keycloak-documentation/pull/1709/files . So either this change in the "fapi-support.adoc" should be added as well to this PR or the sentence : `Details in the link:{adapterguide_link}#_open-banking-uk[{adapterguide_name}]` should be removed.

Reply:
I dropped the link, and then you can add it back again in your PR with the docs update if you want? Does that work?